test/files/GenerateProto.md

```diff
@@ -0,0 +1,3 @@
+Folder: TestGrpc
+
+protoc --proto_path=TestGrpc --go_out=testGrpc --go_opt=paths=source_relative TestGrpc/Example.proto --descriptor_set_out=TestGrpc/example.pb --include_imports
```

Comment:
Can you make this a README or a Makefile? Also I believe the paths aren't correct at the moment. 🤔